packager/packager.cc

```diff
@@ -218,11 +218,11 @@ class PackageInfoImpl final : public sorbet::core::packages::PackageInfo {
             if (!exports.empty()) {
                 exportLoc = exports.front().fqn.loc.beginPos() - "export "sv.size() - 1;
             } else {
-                exportLoc = info.loc.endPos() - "end"sv.size() - 1;
+                exportLoc = loc.endPos() - "end"sv.size() - 1;
```

Comment:
The issues with the previous one were that we want to calculate the `loc` from _this_ package, not from `info` (which is the package we're importing.)